tests/test_ilastik/test_applets/dataSelection/testOpDataSelection.py

```diff
@@ -864,17 +861,119 @@ class TestOpDataSelection_PrecomputedChunks:
     }
 
     @pytest.fixture
-    def op_and_dataset(self, graph, monkeypatch) -> (OpDataSelection, MultiscaleUrlDatasetInfo):
-        _ = MockRemoteDataset(monkeypatch, self.MOCK_DATASET_URL, self.INFO_JSON, self.CHUNKS)
+    def datasetInfo(self, monkeypatch):
+        mock_precomputed_requests(monkeypatch, self.MOCK_DATASET_URL, self.INFO_JSON, self.CHUNKS)
+        return MultiscaleUrlDatasetInfo(url=self.MOCK_DATASET_URL)
+
+    @pytest.fixture
+    def op(self, graph, monkeypatch, datasetInfo) -> OpDataSelection:
+        op = OpDataSelection(graph=graph)
+        op.WorkingDirectory.setValue(os.getcwd())
+        op.ActiveScale.setValue(DEFAULT_LOWEST_SCALE_KEY)
+        op.Dataset.setValue(datasetInfo)
+        return op
+
+    def test_load_precomputed_chunks_over_http(self, op):
+        loaded_scale0 = op.Image[:].wait()
+        assert numpy.allclose(loaded_scale0, self.IMAGE_SCALED)
```

Comment:
```suggestion
        numpy.testing.assert_allclose(loaded_scale0, self.IMAGE_SCALED)
```

I think this produces better error messages.

Reply:
Looks like it behaves slightly differently. numpy.testing.allclose fails the test: `AssertionError:  Not equal to tolerance rtol=1e-07, atol=0 (shapes (1, 1, 1, 10, 12), (1, 10, 12) mismatch)`, so numpy.allclose has been ignoring a mismatch of singleton axes all this time

Reply:
Wow, did not expect that...

Excerpt from NumPy docs:

> Returns True if two arrays are **element-wise equal** within a tolerance.